plugin/build.gradle

```diff
@@ -192,14 +192,6 @@ integTest {
     // Set this to true this if you want to see the logs in the terminal test output.
     // note: if left false the log output will still show in your IDE
     testLogging.showStandardStreams = true
-
-    // Exclude integration test for search index tool for multiple nodes
-    // because we cannot get concrete exception message in multi-node cluster
```

Comment:
Looks like there's a note why this section is commented out. How are we solving this issue then?

Reply:
We are bringing back IT search index tool for multi node cluster.

Reply:
After merging https://github.com/opensearch-project/ml-commons/issues/2443 the issue has been resolved.